.github/workflows/pr-test.yml

```diff
@@ -23,6 +23,10 @@ jobs:
     env:
       BASE_SHA: ${{ github.event.pull_request.base.sha }}
       HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+      # If we don't do this the built files will end up in
+      # `node_modules/@mdn/yari/client/build/` and we don't want that
+      # to get pushed into the cache.
+      BUILD_OUT_ROOT: /tmp/build
```

Comment:
This should also work, see [GitHub docs](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow)